### ISSUE TITLE
Pale Moon

### ISSUE THREAD

**Issue by Ghostwalker9** [nicu03/dark-youtube-theme#10]
Dark Youtube Theme had been working in Pale Moon until version 25.8.1. As popular as PM is now just thought I'd mention that it still works as long as the PM GUID: 8de7fcbb-c55c-4fbe-bfc5-fc555c87dbc4 is inserted into the install.rdf and the minimum version <em:minVersion>26.0</em:minVersion> is set to 25.0. Thank you for this very nice add-on.

**Comment by nicu03** [comment]
i don't think this is an issue; when i uploaded the latest version, by mistake, i've set the max version to 35 instead of 46 
Just corrected it, don't know how long will it take for the update to be visible.
thanks for letting me know!

**Comment by nicu03** [comment]
@Ghostwalker9 any updates? can i close this ?
